src/shared/services/commandInterpreterHelper.ts

```diff
@@ -219,7 +219,9 @@ const availableCommands = [
         const cleanDbName = unescapeCypherIdentifier(normalizedName)
 
         const dbMeta = getDatabases(store.getState()).find(
-          (db: any) => db.name.toLowerCase() === cleanDbName
+          (db: Database) =>
+            db.name.toLowerCase() === cleanDbName ||
+            db.aliases?.find(alias => alias.toLowerCase() === cleanDbName)
```

Comment:
How about reuse `findDatabaseByNameOrAlias` method from `dbMetaDuck`?

Reply:
So, we can just check if `findDatabaseByNameOrAlias` returns `undefined`.

Reply:
Good catch